### PR TITLE
Fix SQL error opening Manage Users page with MSSQL

### PR DESCRIPTION
**Pull request by dregad**
[Microsoft][ODBC Driver 17 for SQL Server][SQL Server]No column name was specified for column 2 of 'tmp'.

Adding aliases to subquery's columns.

Fixes [#33372](https://www.mantisbt.org/bugs/view.php?id=33372)

### CODE REVIEW

**Comment by vboctor** [comment]
Should the having statement use `mycount` rather than `count(*)`?

**Comment by dregad** [comment]
> Should the having statement use `mycount` rather than `count(*)`?

That won't work actually...

### MySQL
```
mysql> SELECT lower(email) email, id, username FROM mantis_user_table 
WHERE lower(email) IN (
  SELECT email  FROM ( SELECT lower(email) email, COUNT(*) FROM mantis_user_table  
  GROUP BY lower(email) HAVING mycount > 1 ) tmp 
) ORDER BY lower(email), username;
ERROR 1054 (42S22): Unknown column 'mycount' in 'having clause'
```

### PostgreSQL
```
bugtracker=# SELECT lower(email) email, id, username FROM mantis_user_table 
bugtracker-# WHERE lower(email) IN (
bugtracker(#   SELECT email  FROM ( SELECT lower(email) email, COUNT(*) FROM mantis_user_table  
bugtracker(#   GROUP BY lower(email) HAVING mycount > 1 ) tmp 
bugtracker(# ) ORDER BY lower(email), username;
ERROR:  column "mycount" does not exist
LINE 4:   GROUP BY lower(email) HAVING mycount > 1 ) tmp 
```

**Comment by vboctor** [comment]
Maybe because you don't have AS mycount in the query?

**Comment by dregad** [comment]
Sorry I copy/pasted the SQL from the wrong branch 😊 let me try again

**Comment by dregad** [comment]
So MySQL is OK with the *AS* but PostgreSQL still chokes

```
bugtracker=# SELECT lower(email) email, id, username FROM mantis_user_table WHERE lower(email) IN ( SELECT email  FROM ( SELECT lower(email) AS email, COUNT(*) AS mycount FROM mantis_user_table GROUP BY lower(email) HAVING mycount > 1 ) tmp ) ORDER BY lower(email), username;
ERROR:  column "mycount" does not exist
LINE 1: ...OM mantis_user_table GROUP BY lower(email) HAVING mycount > ...
```

**Comment by dregad** [comment]
I can't actually test it, but most likely this wouldn't work with MSSQL either according to some quick research I did.

Further reading https://dba.stackexchange.com/questions/21965/why-are-queries-parsed-in-such-a-way-that-disallows-the-use-of-column-aliases-in/

**Review by vboctor — APPROVED** [review]
